Aktualizace slovníku učitelů v UcitelPickerPage.xaml.cs

OndrejZmatlik opened the pull request:
Aktualizován slovník učitelů v souboru UcitelPickerPage.xaml.cs. Změněny kódy několika učitelů:
- "Bezstarosti Pavel" z "PB" na "BZ"
- "Filip Nathaniel" z "NF" na "FI"
- "Hloušek Jiří" z "HJ" na "HU"
- "Lang Matěj" na "Lang Jan" s kódem "LA"
- "Petera Ondřej" z "OP" na "PT"
- "Šolc Miloš" z "MS" na "SO"
- "Ženíšková Eva" z "EZ" na "ZN"

Přidáni noví učitelé:
- "Jungová Miluše" s kódem "JU"
- "Langová Kateřina" s kódem "LV"
- "Langová Martina" s kódem "LG"
- "Lenc Tomáš" s kódem "LE"
- "Silná Petra" s kódem "PS"

OndrejZmatlik committed:
Aktualizace slovníku učitelů v UcitelPickerPage.xaml.cs

Aktualizován slovník učitelů v souboru UcitelPickerPage.xaml.cs.
Změněny kódy několika učitelů:
- "Bezstarosti Pavel" z "PB" na "BZ"
- "Filip Nathaniel" z "NF" na "FI"
- "Hloušek Jiří" z "HJ" na "HU"
- "Lang Matěj" na "Lang Jan" s kódem "LA"
- "Petera Ondřej" z "OP" na "PT"
- "Šolc Miloš" z "MS" na "SO"
- "Ženíšková Eva" z "EZ" na "ZN"

Přidáni noví učitelé:
- "Jungová Miluše" s kódem "JU"
- "Langová Kateřina" s kódem "LV"
- "Langová Martina" s kódem "LG"
- "Lenc Tomáš" s kódem "LE"
- "Silná Petra" s kódem "PS"

AlmoAsdwak reviewed (commented):
- "Jungová Miluše" s kódem "JU"
- "Langová Kateřina" s kódem "LV"
- "Langová Martina" s kódem "LG"
- "Silná Petra" s kódem "PS"
- neučí

a chybí to ještě v UcitelPickerPage.xaml

OndrejZmatlik committed:
Aktualizace UcitelPickerPage pro dynamický seznam učitelů

Odstraněn pevně zakódovaný seznam učitelů v UcitelPickerPage.xaml.
Přidán using pro System.Linq v UcitelPickerPage.xaml.cs.
Aktualizován event handler Button_Clicked:
- Nastavení ItemsSource na null na začátku metody.
- Aktualizace slovníku učitelů (odstranění a přidání položek).
- Nastavení ItemsSource na klíče slovníku učitelů převedené na seznam.
